README.md

```diff
@@ -1 +1,47 @@
 # Git Assignment - <Your GitHub Username>
```

Comment:
Remember to add your name

Reply:
Where do I suppose to add it? In the comment when I do the commit?

Reply:
Supposedly you would do it in the readme.md where this line of code exist. But your work is approved so you don't necessarily need to change this.